src/lib/packet.h

```diff
@@ -582,6 +601,12 @@ typedef struct pgp_elgamal_sig_t {
     BIGNUM *s;
 } pgp_elgamal_sig_t;
 
+/** pgp_ecc_signature_t */
+typedef struct pgp_ecc_sig_t {
+    BIGNUM *r;
+    BIGNUM *s;
+} pgp_ecc_sig_t;
```

Comment:
actually, we don't need new structure. This is the same as pgp_dsa_sig_t and can be used for DSA, EdDSA and ECDSA (it's just 2 bignums in the end)